server/features/extraction/extract_text.py

```diff
@@ -36,6 +38,30 @@ def extract_text(file_name: str, file_type: str, file: bytes) -> Document:
         semantic_identifier=file_name
     )
 
+def extract_texts_from_image(file_name: str, file: bytes) -> Document:
+    """
+    Summary
+    -------
+    extract the text from an image
+
+    Parameters
+    ----------
+    file_name (str): the name of the file
+    file (bytes): the file
+
+    Returns
+    -------
+    document (Document): the parsed document
+    """
+    with PyTessBaseAPI() as ocr:
+        with Image.open(file) as img:
+            sections = [ocr.image_to_text(img)]
+
+    return Document(
+        id=str(uuid4()),
+        sections=sections,
+        semantic_identifier=file_name
+    )
```

Comment:
I think we should abstract this out into it's own file. We will probably have a `extract_document_from_image` and a `extract_text_from_image`. The former is for creating an embedding and the latter will be for querying with an image.

Reply:
I'll probably split it into `extract_from_image.py` and then rename the current one to `extract_from_pdf.py`.

Reply:
ill work on that